### ISSUE TITLE
[Bug] PDF Footer search box appear

### ISSUE THREAD

**Issue by JM2K69** [dotnet/docfx#10414]
**Describe the bug**
In my **docfx.json** file
```
      "pdfCoverPage":"./pdf/cover.html",
      "pdfFooterTemplate":"./pdf/footer.html"
```

**To Reproduce**
1. **pdfFooterTemplate** content
```
<div style="width: 100%; font-size: 12px;">
  <div style="float: right; padding: 0 2em">
    <span class="pageNumber"></span> / <span class="totalPages"></span>
  </div>
</div>
```
3. Build the site with docfx 2.78.1
pdf file have a searchbox on the footer

![Image](https://github.com/user-attachments/assets/34a8f8a9-95dc-423b-b104-c9ad35a38fc6)

The cover page works but not the **footer**.

**Context (please complete the following information):**
- OS: Windows 11
- Docfx version: [2.78.1]

**Additional context**

**Comment by JM2K69** [comment]
I try a new project the bug appear when the setting `enableSearch` is enabled in **docfx.json**

1. **_enableSearch disable**
![Image](https://github.com/user-attachments/assets/e26626ae-10ca-4ca1-b8d3-9d86a1ee5e36)

2. **_enableSearch enable**
![Image](https://github.com/user-attachments/assets/f274cc00-b5e3-4f91-b081-2549760e1334)
 @filzrev

**Comment by JM2K69** [comment]
Still present in 2.78.2.

**Comment by filzrev** [comment]
On current implementation.
It seems `pdfHeaderTemplate`/ `pdfFooterTemplate` don't support HTML files that are generated from markdown.
([cover page supports HTML file generated from markdown](https://dotnet.github.io/docfx/docs/pdf.html?q=cover#pdffootertemplate) )

For `pdfHeaderTemplate`/ `pdfFooterTemplate`.
It need to specify `HTML Fragment` file path.
And need to copy HTML files as [`resource` files](https://dotnet.github.io/docfx/reference/docfx-json-reference.html?q=resource#resource).

**Comment by JM2K69** [comment]
I will test with HTML ressources files.

**Comment by JM2K69** [comment]
@filzrev  It works but when I used Image in cover.html and an other one in footer [#10416](https://github.com/dotnet/docfx/pull/10416) the image on the footer is used in cover too Like this issue but I don't know if this merge is include in the version 2.78.2
ping @yufeih

**Comment by filzrev** [comment]
> It works but when I used Image in cover.html and an other one in footer https://github.com/dotnet/docfx/pull/10416 the image on the footer is used in cover too Like this issue but I don't know if this merge is include in the version 2.78.2

It's expected to be fixed on v2.78.2.
I've tested with provided `cover.html`/`footer.html` files. but it can't be reproduced. (Specified image is displayed on cover page)

Is it able to provide files that can reproduce reported issue?

**Comment by JM2K69** [comment]
> > It works but when I used Image in cover.html and an other one in footer [#10416](https://github.com/dotnet/docfx/pull/10416) the image on the footer is used in cover too Like this issue but I don't know if this merge is include in the version 2.78.2
> 
> Il est prévu qu’il soit corrigé sur la v2.78.2. J’ai testé avec les fichiers / fournis. Mais il ne peut pas être reproduit. (L’image spécifiée s’affiche sur la page de couverture)`cover.html``footer.html`
> 
> Est-il en mesure de fournir des fichiers capables de reproduire le problème signalé ?

@filzrev yes I can reproduce clone my repository [JM2K69/Docfx_Fix_Header](https://github.com/JM2K69/Docfx_Fix_Header) and run `docfx --server docfx.json -p 80`

```
docfx --serve .\docfx.json -p 80
Searching built-in plugins in directory C:\Users\JM2K69\.dotnet\tools\.store\docfx\2.78.2\docfx\2.78.2\tools\net9.0\any\...
Post processor ExtractSearchIndex loaded.
7 plug-in(s) loaded.
Building 3 file(s) in ConceptualDocumentProcessor(BuildConceptualDocument=>ValidateConceptualDocumentMetadata)...
Building 3 file(s) in ResourceDocumentProcessor(ValidateResourceMetadata)...
Building 2 file(s) in TocDocumentProcessor(BuildTocDocument)...
Applying templates to 8 model(s)...
XRef map exported.
Extracting index data from 3 html files
Using Node.js v22.11.0 executable.

     toc.pdf ---------------------------------------- 100%
docs\toc.pdf ---------------------------------------- 100%

Serving "D:\docfx\2.78.2\_site" on http://localhost:80
Press Ctrl+C to shut down
```

**PDF** is like this 

![Image](https://github.com/user-attachments/assets/ba2b011f-6c7e-4c34-8f1b-1c28b30098a7)

**Comment by filzrev** [comment]
Thanks for providing reproduceable code.
I can also reproduce reported issues.

On my environment. I'm using following configs (without `./`)
And in this case it works as expected.

```
      "pdfCoverPage":"pdf/cover.html",
      "pdfFooterTemplate": "pdf/footer.html"
```
